content/applications/finance/accounting/bank/setup/outstanding_accounts.rst

```diff
@@ -0,0 +1,36 @@
+====================
+Outstanding Accounts
+====================
+
+In Odoo, before being recorded in your bank account, payments are registered through transitory
+accounts, named Outstanding Accounts.
```

Comment:
```suggestion
accounts, named :guilabel:`Outstanding Accounts`.
```
@jcs-odoo Is this guideline already in place? (send your mail)

Reply:
> @jcs-odoo Is this guideline already in place? (send your mail)

yes, but still not online, sorry :(